docs/changelog.rst

```diff
@@ -9,6 +9,7 @@ Changelog
 * :feature:`1793` The PnL report can now be generated with the average cost basis accounting method.
 * :bug:`-` Transfers between tracked accounts will now have a correct label in the UI.
 * :bug:`5038` Premium users with big databases should no longer see the error: "Upload data to server died with exception: database plaintext is locked"
+* :bug:`-` Price oracle penalty mechanism now uses the correct penalty timeout
```

Comment:
I am sorry but @prettyirrelevant here made a mistake. Even if this was a bug fix this is not user facing as it's just a timeout value.

But it's not even. This feature is not yet released so no need to do this. Please amend the commit and remove this.

Reply:
No problem, updated